changelogs/fragments/9573-iocage-inventory-sudo.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - iocage inventory plugin - the new parameter ``sudo`` of the plugin lets the command ``iocage list -l`` to run as root on the iocage host. This is needed to get the IP4 of a running DHCP jail (https://github.com/ansible-collections/community.general/issues/9572).
```

Comment:
Nit picking: it's not wrong, however the form with the `v` is more commonly used.
```suggestion
  - iocage inventory plugin - the new parameter ``sudo`` of the plugin lets the command ``iocage list -l`` to run as root on the iocage host. This is needed to get the IPv4 of a running DHCP jail (https://github.com/ansible-collections/community.general/issues/9572).
```

Reply:
Also please add the PR's URL next to the issue URL. Thanks!